packages/salesforcedx-sobjects-faux-generator/src/generator/fauxClassGenerator.ts

```diff
@@ -25,6 +25,39 @@ export class FauxClassGenerator {
   private STANDARDOBJECTS_DIR = 'standardObjects';
   private CUSTOMOBJECTS_DIR = 'customObjects';
 
+  // the empty string is used to represent the need for a special case
+  // usually multiple fields with specialized names
+  private static typeMapping: Map<string, string> = new Map([
```

Comment:
If it is a specialized field, we should designate a proper sentinel to mark it. Using an empty '' string is bad. Are you doing special checks somewhere to check for '' strings?

Reply:
Is this only used at 

```
private getTargetType(describeType: string): string {
 +    const gentype = FauxClassGenerator.typeMapping.get(describeType) as string;
 +    return gentype ? gentype : this.capitalize(describeType);
 +  }
``